tests/common/utilities.py

```diff
@@ -488,7 +488,10 @@ def compare_crm_facts(left, right):
     for k, v in list(left['resources'].items()):
         lv = v
         rv = right['resources'][k]
-        if lv['available'] != rv['available'] or lv['used'] != rv['used']:
+
+        # For Cisco-8000 devices, hardware counters are statistical-based with +/- 1 entry tolerance.
```

Comment:
this will remove the check for all vendors and add only cisco specific check for all. This should not be done this way. You need to add the check to only take into effect when running on Cisco specific devices